cmd/bootstrap.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/manifoldco/go-base64"
+	"github.com/urfave/cli"
+
+	"bufio"
+	"path/filepath"
+
+	"github.com/manifoldco/torus-cli/gatekeeper/bootstrap"
+	"github.com/manifoldco/torus-cli/identity"
+)
+
+const (
+	// GlobalRoot is the global root of the Torus config
+	GlobalRoot = "/etc/torus"
+
+	// EnvironmentFile is the environment file that stores machine information
+	EnvironmentFile = "token.environment"
+)
+
+func init() {
+	bootstrap := cli.Command{
+		Name:     "bootstrap",
+		Usage:    "Bootstrap a new machine using Torus Gatekeeper",
+		Category: "SYSTEM",
+		Flags: []cli.Flag{
+			authProviderFlag("Auth provider for bootstrapping", true),
+			machineFlag("Machine name to bootstrap", false),
+			urlFlag("Gatekeeper URL for bootstrapping", true),
+			orgFlag("Org the machine will belong to", false),
+			roleFlag("Role the machine will belong to", false),
```

Comment:
It would be cool to think about how the machine name, org, and role could be handled by the auth mechanism itself, on the gatekeeper side. Not for this pr though, of course :)